tests/utils/pod.go

```diff
@@ -146,7 +146,7 @@ func podStatus(clientSet *kubernetes.Clientset, podName, namespace string, statu
 			}
 			return false, err
 		}
-		ginkgo.By("Checking POD phase: " + string(pod.Status.Phase))
+		fmt.Fprintf(GinkgoWriter, "INFO: Checking POD phase: %s\n", string(pod.Status.Phase))
```

Comment:
We drop the dot import so you'd need "ginkgo.GingkoWriter" here (sorry, I know that sort of annoying), we could name the import if we want but no real value there IMO

Reply:
No, that is good, I forgot to run make test-lint on this patch, and I should have.